### PR TITLE
Retry guidance: Rewording Cosmos DB retry guidance

### PR DESCRIPTION
**Pull request by ealsur**
I work on the Cosmos DB SDK team, the current retry guidance is not completely correct and it's covering only one case.

We have a more detailed document that guides customers to create resilient applications covering different retry scenarios and would be best to link to that one.

### CODE REVIEW

**Comment by PRMerger4** [comment]
@ealsur : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by j82w — APPROVED** [review]

**Comment by PRMerger15** [comment]
@ealsur : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by kushagraThapar — APPROVED** [review]

**Review by ckittel — APPROVED** [review]
LGTM!  Thanks for reducing the partial duplication and improving the quality at the same time.

**Comment by ealsur** [comment]
#sign-off

**Comment by PRMerger5** [comment]
@ealsur: I'm sorry - only the author of this article, @EdPrice-MSFT, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

File: docs/best-practices/retry-service-specific.md [hunk 1]
```diff
@@ -77,51 +77,11 @@ Cosmos DB is a fully managed multi-model database that supports schemaless JSON
...
-
-```csharp
-CosmosClient cosmosClient = new CosmosClient("connection-string", new CosmosClientOptions()
-{
-    MaxRetryAttemptsOnRateLimitedRequests = 5,
-    MaxRetryWaitTimeOnRateLimitedRequests = TimeSpan.FromSeconds(15)
-});
-
-```
+The Cosmos DB SDKs automatically retry on certain error conditions, and user applications are encouraged to have their own retry policies. Please see the [guide to designing resilient applications with Azure Cosmos DB SDKs](/azure/cosmos-db/sql/conceptual-resilient-sdk-applications) for a complete list of error conditions and when to retry.
```
Comment (ShannonLeavitt) [inline docs/best-practices/retry-service-specific.md:80]:
```suggestion
The Cosmos DB SDKs automatically retry on certain error conditions, and user applications are encouraged to have their own retry policies. See the [guide to designing resilient applications with Azure Cosmos DB SDKs](/azure/cosmos-db/sql/conceptual-resilient-sdk-applications) for a complete list of error conditions and when to retry.
```
Fixing minor style issue

**Comment by ShannonLeavitt** [comment]
@ckittel - I made a few edits for minor style issues. Could you review and add a `#sign-off` if the commits are ready to merge?

**Review by ckittel — APPROVED** [review]

**Comment by PRMerger9** [comment]
@ckittel: I'm sorry - only the author of this article, @EdPrice-MSFT, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Pull request merged**